src/modules/rangeproof/main_impl.h

```diff
@@ -231,17 +231,23 @@ int secp256k1_pedersen_blind_generator_blind_sum(const secp256k1_context* ctx, c
 
 int secp256k1_rangeproof_info(const secp256k1_context* ctx, int *exp, int *mantissa,
  uint64_t *min_value, uint64_t *max_value, const unsigned char *proof, size_t plen) {
+    secp256k1_rangeproof_header header;
     size_t offset;
-    uint64_t scale;
     ARG_CHECK(exp != NULL);
     ARG_CHECK(mantissa != NULL);
     ARG_CHECK(min_value != NULL);
     ARG_CHECK(max_value != NULL);
     ARG_CHECK(proof != NULL);
     offset = 0;
```

Comment:
`secp256k1_rangeproof_getheader_impl` took offset as an input, but `header_parse` doesn't. Remove it?

Reply:
`header_parse` does take it as an input.

Reply:
`header_parse` does take it as an input.

Reply:
`header_parse` does take it as an input.

Reply:
Sorry that was  ambiguous. `secp256k1_rangeproof_getheader_impl` reads `*offset` (but is always called with `*offset = 0` and `header_parse` doesn't.

Reply:
Ah I understand! `header_parse` treats this field as write-only, so there is no longer any need to zero- initialize it. Ok, I will delete this line.